### DISCUSSION TITLE
Override option for legend 'values'

### DISCUSSION THREAD

**Discussion by rsporsche** [grafana/grafana#70296] [Dashboards, Panels & Visualizations]
I have a number of series on a single visualisation e.g. max, mean, and min of a metric using fill below from max to min. 

I would like to show the "*last" value in the legend for all three series but only the "max" value for the max series and the "min" value for the min series. 

There doesn't seem to be an override option for the legend values.

That's just one example. I have other visualisations with more series where the issue becomes more acute.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82665. Feel free to continue the discussion around this there. Thank you!
